lib/fog/libvirt/models/compute/volume.rb

```diff
@@ -118,6 +118,9 @@ def split_size_unit(text)
             # if text is an integer, match will fail
             size    = text
             unit    = 'G'
+          elsif text.blank?
```

Comment:
I think it's better to do `text.to_i` because it yields 0 for both nil or empty string and saves us an "elseif" branch reducing complexity.